packages/request-client.js/src/api/request.ts

```diff
@@ -692,4 +701,18 @@ export default class Request {
 
     return this.getData();
   }
+
+  /**
+   * Enables the payment detection
+   */
+  public enablePaymentDetection(): void {
```

Comment:
this mean to get the balance you need to do `request.enablePaymentDetection()`, then `request.refresh()`? 
Would it make sense to add a utility like `loadBalance()`? (thinking out loud, no opinion)